### ISSUE TITLE
optimizer and scheduler

### ISSUE THREAD

**Issue by inconnu11** [ming024/FastSpeech2#112]
Hi, why do you update learning rate before optimizer.step() in [code](https://github.com/ming024/FastSpeech2/blob/master/model/optimizer.py#L22-L24)? Should not we conduct optimizer.step() fist and then conduct scheduler.step()? Or is there some other consideration?

**Comment by aidosRepoint** [comment]
> Hi, why do you update learning rate before optimizer.step() in [code](https://github.com/ming024/FastSpeech2/blob/master/model/optimizer.py#L22-L24)? Should not we conduct optimizer.step() fist and then conduct scheduler.step()? Or is there some other consideration?

Hi! Have you found an answer for your question?
